docs/sync-web-site.sh

```diff
@@ -38,12 +38,17 @@ fi
 if [ $BRANCH == "main" ] && [ "$QUARKUS_RELEASE" == "true" ]; then
```

Comment:
running this script fails as it uses git@ based format to clone.

If I change it to https://github.com/quarkusio/quarkusio.github.io it just works.

Reply:
thats probably old issue so not related to this one - worth fixing in separate pr IMO though.

Reply:
yes. open a separate issue.